op-batcher/metrics/metrics.go

```diff
@@ -481,9 +493,9 @@ func (m *Metrics) ClearAllStateMetrics() {
 	m.pendingBlocksBytesCurrent.Set(0)
 }
 
-// estimateBatchSize returns the estimated size of the block in a batch both with compression ('daSize') and without
+// EstimateBatchSize returns the estimated size of the block in a batch both with compression ('daSize') and without
 // ('rawSize').
-func estimateBatchSize(block *types.Block) (daSize, rawSize uint64) {
+func EstimateBatchSize(block *types.Block) (daSize, rawSize uint64) {
```

Comment:
Now that this function is not only used within the metrics package, but is also used in the context of the throttling logic, I'm wondering if we want to move it to another package.

Reply:
Because it is used within the metrics package, I think it's ok to stay. Otherwise we will be fighting import cycles. The proper long-term solution is to move all business logic (`pendingDABytes` 👀) out of the `metrics` package.